aten/src/TH/generic/THTensorEvenMoreMath.cpp

```diff
@@ -67,6 +67,26 @@ void THTensor_(nonzero)(THLongTensor *subscript, THTensor *tensor)
   delete [] idx;
 }
 
+accreal THTensor_(sumall)(THTensor *tensor)
```

Comment:
why do you need these?

Reply:
We are using them in THTensor_(maskedSelectBool) for resizing the result tensor.
Example with byte tensor: https://github.com/pytorch/pytorch/blob/master/aten/src/TH/generic/THTensorEvenMoreMath.cpp#L137